akka-actor/src/main/scala/akka/event/LoggerMailbox.scala

```diff
@@ -43,6 +43,5 @@ private[akka] class LoggerMailbox(owner: ActorRef, system: ActorSystem)
         envelope = dequeue
       }
     }
-    super.cleanUp(owner, deadLetters)
```

Comment:
@patriknw @ktoso in fact it is really simple:

1. What is it that call to super does? Sends remaining messages to dead letters.
2. What dead letters does? Sends them to the logger but wait, the logger is dead

If (1) happens; how is it possible to send messages to super if they queue was just drained?
if (1) happens then (2) happens, does it make sense to send to dead-letters to later send to a dead logger?